contrib/automation_tests/orbit_load_presets.py

```diff
@@ -0,0 +1,160 @@
+"""Apply two presets in Orbit using pywinauto.
+
+Before this script is run there needs to be a gamelet reserved and
+"hello_ggp_standalone" has to be started. Two presets named
+draw_frame_in_hello_ggp_1_52.opr and ggp_issue_frame_token_in_hello_ggp_1_52
+(hooking the functions DrawFrame and GgpIssueFrameToken) need to exits in the
+preset folder.
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test it needs
+to by run from  64 bit python.
+"""
+import logging
+import time
+from absl import app
+import pywinauto
+from pywinauto.application import Application
+
+
+def WaitForOrbit():
+  while True:
+    try:
+      Application(backend='uia').connect(title_re='orbitprofiler')
+      break
+    except pywinauto.findwindows.ElementNotFoundError:
+      pass
+
+
+def ConnectToGamelet(application):
+  """Connect to the first gamelet in the start dialog.
+
+  Args:
+    application: pywinauto application object for Orbit.
+  """
+  # Connect to the first (and only) gamelet.
+  dialog = application.window(title_re='orbitprofiler')
+  dialog.set_focus()
+  # DataItem0 contains the name of the first gamelet.
+  dialog.DataItem0.click_input()
+  dialog.OK.click_input()
+  logging.info('Connected to Gamelet.')
+  # Wait until the service is deployed and the main window is opened.
+  # Waiting for the window does not really work since there is a progress dialog
+  # with an identical title popping up in between. So we just sleep.
+  time.sleep(15)
```

Comment:
couldn't you instead wait, until the dialog disappears?

Reply:
There is a sequence of windows popping up during the deployment. I think the right way  would be to query an element from the main window in a try/except block, loop that forever or until a timeout happens and break from the loop when the element from the main window shows up. Pretty much as I do in WaitForOrbit.

I'll refactor this when I see this failing for the first time.

Reply:
Fine for me. Thanks for clarifying.